tensorflow_recommenders_addons/dynamic_embedding/python/ops/redis_table_ops.py

```diff
@@ -59,7 +59,7 @@ class RedisTable(LookupInterface):
 
   default_redis_params = {
       "redis_connection_mode":
-          1,  # ClusterMode = 0, SentinelMode = 1, StreamMode = 2
+          1,  # ClusterMode = 0, SentinelMode = 1, StandaloneMode = 2
```

Comment:
What's the different between StreamMode and StandaloneMode, Will canceling StreamMode cause forward compatibility issues?

Reply:
StreamMode has never been implemented before, so selecting this parameter will exit the program. It was just a failed product of my idea, and now it's just gone.